src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    public static String getRandomColor() {
+        int index = new Random().nextInt(Color.values().length);
```

Comment:
Make Random a class-level variable.
![image](https://github.com/mate-academy/jv-oop-advanced/assets/20326858/9d2eba76-bca9-4c24-8154-355660ca75af)

Reply:
not fixed

Reply:
not fixed